keras_retinanet/models/retinanet.py

```diff
@@ -148,31 +195,78 @@ def num_anchors(self):
 
 
 def default_submodels(num_classes, anchor_parameters):
+    """ Create a list of default submodels used for object detection.
+
+    The default submodels contains a regression submodel and a classification submodel.
+
+    Args
+        num_classes       : Number of classes to use.
+        anchor_parameters : Struct that defines how the anchors should be made.
+
+    Returns
+        A list of tuple, where the first element is the name of the submodel and the second element is the submodel itself.
+    """
     return [
         ('regression', default_regression_model(anchor_parameters.num_anchors())),
         ('classification', default_classification_model(num_classes, anchor_parameters.num_anchors()))
     ]
 
 
 def __build_model_pyramid(name, model, features):
+    """ Builds pyramid for a single submodel.
```

Comment:
More accurately applies the submodel to each FPN level. I wouldn't necessarily call this building a pyramid or constructing it, merely applying over an existing pyramid of features. Perhaps the function signature is misleading?

Reply:
:+1: